javascript/Game.js

```diff
@@ -22,6 +24,26 @@ class Game {
   declareWinner() {
   }
 
-}
+  createCategories() {
+    let categoryKeys = Object.entries(data.categories);
+      let randoms = [];
+        for (var i = 0; i < categoryKeys.length; i++) {
+
+          let rand = categoryKeys[Math.floor(Math.random() * categoryKeys.length)]
+            if (randoms.indexOf(rand) === -1) {
```

Comment:
You can use `.includes()` here instead of checking if the index is -1 (Again, just slightly more english/readable)